examples/01-advanced-postprocessing/Touchstone_Management.py

```diff
@@ -30,9 +28,9 @@
 
 ###############################################################################
 
-from pyaedt.generic.TouchstoneParser import *
-import numpy as np
 import matplotlib.pyplot as plt
+import numpy as np
+from pyaedt.generic.TouchstoneParser import *
```

Comment:
Would prefer to avoid `*` imports.  @jleonatti, is there a flake8 standard for that?  I'd like to enforce it.

Reply:
> I'd like to enforce it.

Just not in this PR.  We can do it in a follow-up PR.

Reply:
@akaszynski yes it would directive F403. I can add then in a follow-up PR.